packages/charts/src/charts/choropleth/choropleth-chart.component.tsx

```diff
@@ -128,12 +129,29 @@ export const ChoroplethChart: FC<Props> = ({
     THREE_DIMENSION_PROJECTIONS.includes(projection) ? 'rotate' : 'translate'
   );
 
-  useZoom(svgElement, setProjectionState, projectionScale);
+  const { zoomDirection } = useZoom(
+    svgElement,
+    setProjectionState,
+    projectionScale
+  );
 
   useEffect(() => {
     if (dragged && themeTooltipSettings.enabled) hideTooltip();
   }, [dragged]);
 
+  const getCursorType = () => {
```

Comment:
Please, consider moving it out of render function and make it `pure` to avoid redeclaring for each re-render for eg. during drag event.